code/modules/events/event.dm

```diff
@@ -190,3 +192,10 @@
 
 	setup()
 	..()
+
+//Called after something followable has been spawned by an event
+//Provides ghosts a follow link to an atom if possible
+//Only called once.
+/datum/event/proc/announce_to_ghosts(atom/atom_of_interest)
+	if(atom_of_interest)
+		notify_ghosts("[name] has an object of interest: [atom_of_interest]!", title = "Something's Interesting!", source = atom_of_interest, action = NOTIFY_FOLLOW)
```

Comment:
Yeah, aware of this one; it's this way because it's meant to be fairly generic---not every event will have a name or atom of interest that is identical---as such, some generalization is kinda required for this to be utilized broadly.